NickvisionCavalier.GNOME/Controls/ColorBox.cs

```diff
@@ -0,0 +1,88 @@
+using NickvisionCavalier.GNOME.Helpers;
+using NickvisionCavalier.Shared.Models;
+using System;
+using System.Runtime.InteropServices;
+
+namespace NickvisionCavalier.GNOME.Controls;
+
+public class ColorEventArgs : EventArgs
+{
+    public readonly ColorType Type;
```

Comment:
See property comments above:

They should be `public DataType Name { get; init; }` (`private set` if you access them in other methods of the same class, `init` if you set them only in constructor, a.k.a readonly)